src/webgl/shaders/lighting.glsl

```diff
@@ -141,14 +137,9 @@ vec3 calculateImageSpecular( vec3 vNormal, vec3 vViewPosition ){
 #endif
   // this is to make the darker sections more dark
   // png and jpg usually flatten the brightness so it is to reverse that
-  if(metallic != 0.0){
-   float invertedMetallic = 1.0 - (metallic / 100.0);
-  float mappedValue = mix(1.35, 8.0, (invertedMetallic));
-  return pow(outColor.xyz, vec3(mappedValue));
-  }
-  else{
-    return pow(outColor.xyz, vec3(10.0));
-  }
+ float invertedMetallic = 1.0 - (metallic / 100.0);
+ float mappedValue = mix(1.2, 10.0, invertedMetallic);
+ return pow(outColor.xyz, vec3(mappedValue));
```

Comment:
I notice that as I bright the metalness slider up, in the middle, the shadows dip darker before they finally brighten:

<table>
<tr>
<td>

<img width="408" alt="image" src="https://github.com/processing/p5.js/assets/5315059/a8ce700b-5ab4-497b-9956-9577c4f32761">

</td>
<td>

<img width="411" alt="image" src="https://github.com/processing/p5.js/assets/5315059/c3e9f315-16ea-432a-b670-6886b1bbf90c">

</td>
<td>

<img width="406" alt="image" src="https://github.com/processing/p5.js/assets/5315059/fd091037-06d6-49d9-aa9b-fe746650e035">

</td>
</tr>
</table>

I wonder if this is because we change the specular/diffuse balance linearly, but the power exponentially (since we linearly interpolate the *exponent.*) Maybe we'll avoid this dip if, instead of linearly interpolating the exponent, we linearly mix between to fixed exponents, like this?
```glsl
return mix(
  pow(outColor.xyz, vec3(1.2)),
  pow(outColor.xyz, vec3(10)),
  invertedMetallic
);
```